diffyscan/utils/encoder.py

```diff
@@ -0,0 +1,121 @@
+from .custom_exceptions import EncoderError
+
+
+def to_hex_with_alignment(value: int) -> str:
+    return format(value, "064x")
+
+
+def encode_address(address: str) -> str:
+    number = int(address, 16)
+    return to_hex_with_alignment(number)
+
+
+def encode_bytes32(data: str) -> str:
+    return data.replace("0x", "")
+
+
+def encode_bytes(data: str) -> str:
+    bytes_str = data.lstrip("0x")
+    if not bytes_str:
+        return to_hex_with_alignment(0)
+
+    # untested, taken from here:
```

Comment:
huh?

Reply:
i'm waiting the example of specific contract

Reply:
here: https://sepolia-optimism.etherscan.io/address/0xf49D208B5C7b10415C7BeAFe9e656F2DF9eDfe3B

init tx: https://sepolia-optimism.etherscan.io/tx/0x4c9db6f7c0e2ca4b831bfc67f46f265a1bdac5735738900e42014c88ef361373

Reply:
https://docs.optimism.io/chain/networks#op-sepolia